### ISSUE TITLE
Opening doc link in VS2017 uses "unsupported" internet explorer browser

### ISSUE THREAD

**Issue by kippesp** [HJLebbink/asm-dude#104]
The default documentation URL points to github.com.  Doing this in VS2017 results in a message from github that internet explorer is an unsupported browser.

Workaround is to use local storage/clone of the wiki.

**Comment by HJLebbink** [comment]
Yes I havent found a solution for that in the last three years, thus if someone knows how to open the default webbrowser and direct it to a certain webpage, do let me know!

**Comment by kippesp** [comment]
Sorry for the duplicate--I searched.  If anyone else is interested in my workaround, it is fairly trivial.  Do this in gitbash (or macos or linux) with pandoc installed:
```
# after cloning the wiki
cd doc
for f in *.md; do pandoc -t html --css NICE_STYLE.css "$f" -s -o "${f%.md}"; done
```
The css style made it as attractive as the github wiki--just find a dark theme if you into that kind of UI.
